packages/curve-ui-kit/src/features/market-position-details/BorrowInformation.tsx

```diff
@@ -0,0 +1,161 @@
+import { CardHeader, Box } from '@mui/material'
+import type {
+  Pnl,
+  BorrowAPY,
+  Leverage,
+  CollateralValue,
+  Ltv,
+  TotalDebt,
+  LiquidationRange,
+  BandRange,
+} from '@ui-kit/features/market-position-details/BorrowPositionDetails'
+import { CollateralMetricTooltip } from '@ui-kit/features/market-position-details/tooltips/CollateralMetricTooltip'
+import { LiquidityThresholdTooltip } from '@ui-kit/features/market-position-details/tooltips/LiquidityThresholdMetricTooltip'
+import { PnlMetricTooltip } from '@ui-kit/features/market-position-details/tooltips/PnlMetricTooltip'
+import { t } from '@ui-kit/lib/i18n'
+import { Metric } from '@ui-kit/shared/ui/Metric'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+const dollarUnitOptions = {
+  abbreviate: false,
+  unit: {
+    symbol: '$',
+    position: 'prefix' as const,
+    abbreviate: false,
+  },
```

Comment:
I don't believe you need to redefine the dollar symbol, can't you just use `unit: 'dollar'`?